connectors/module-messageStore/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
Shall we remove multiple gitignore files and mention these in the gitignore file in the root folder?

Reply:
In future, `module-messageStore` will be a separate repository. So will be keeping it for the moment.